ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -33,13 +33,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: installer-rehearse-aws
+- as: installer-rehearse-aws-ipi
```

Comment:
suggest to naming it with cluster profile name, e.g: `installer-rehearse-aws-qe`